server/github.go

```diff
@@ -332,7 +332,7 @@ func (s *Server) createRepoStatus(ctx context.Context, pr *model.PullRequest, st
 }
 
 func (s *Server) waitForStatus(ctx context.Context, pr *model.PullRequest, statusContext string, statusState string) error {
-	ticker := time.NewTicker(5 * time.Second)
+	ticker := time.NewTicker(30 * time.Second) // extended to 30 seconds to go with the secondary rate limit interval of GitHub
```

Comment:
It's a big jump. I think this makes sense with E2E but I don't know if we want that for other case like triggering EE build where the user somehow expects an "immediate" response.

Reply:
Maybe I should hand ticker time in as a parameter?

Reply:
Sounds good to me and thanks for considering!

Reply:
Of course!

Reply:
Thanks! LGTM.